scss/facet-list.scss

```diff
@@ -177,7 +177,8 @@ $facetlist-term-block-height: default !default;
                     
                     div.range-drop {
                         display: flex;
-                        align-items: center;                    
+                        align-items: center;  
+                        line-height: 1;
```

Comment:
Changes the implicit height of the 'From' and 'To' text areas so they center alongside the dropdown buttons a tiny bit better (IMO / possibly subjective).

Reply:
Maybe subjective, but I think Burak said something similar. I'm not always great with those details, so I'll take your word and his for it that this is an improvement.